docs/source/deploypeer/peerchecklist.md

```diff
@@ -200,26 +200,31 @@ Peers leverage the Gossip data dissemination protocol to broadcast ledger and ch
 
 ```
 tls:
-# Require server-side TLS
-enabled:  false
-# Require client certificates / mutual TLS.
-# Note that clients that are not configured to use a certificate will
-# fail to connect to the peer.
-clientAuthRequired: false
-# X.509 certificate used for TLS server
-cert:
-    file: tls/server.crt
-# Private key used for TLS server (and client if clientAuthEnabled
-# is set to true
-key:
-    file: tls/server.key
-# Trusted root certificate chain for tls.cert
-rootcert:
-    file: tls/ca.crt
-# Set of root certificate authorities used to verify client certificates
-clientRootCAs:
-    files:
-      - tls/ca.crt
+    # Require server-side TLS
+    enabled:  false
+    # Require client certificates / mutual TLS for inbound connections.
+    # Note that clients that are not configured to use a certificate will
+    # fail to connect to the peer.
+    clientAuthRequired: false
+    # X.509 certificate used for TLS server
+    cert:
+        file: tls/server.crt
+    # Private key used for TLS server
+    key:
+        file: tls/server.key
+    # rootcert.file represents the trusted root certificate chain used for verifying certificates
+    # of other nodes during outbound connections.
+    # It is not required to be set, but can be used to augment the set of TLS CA certificates
+    # available from the MSPs of each channel’s configuration.
+    rootcert:
+        file: tls/ca.crt
+    # If mutual TLS is enabled, clientRootCAs.files contains a list of additional root certificates
+    # used for verifying certificates of client connections.
+    # It is not required to be set, but can be used to augment the set of TLS CA certificates
```

Comment:
I don't think this is quite right... if you require client auth, and you do not set this, how can you join a peer to a channel? It's only 'not required' once your peer has channels.

Reply:
I think he meant it is not required to be set since the mutual TLS is also not required